nanoGPT/README.md

```diff
@@ -31,19 +36,29 @@ python data/openwebtext/prepare.py
 
 #### 训练
 
-使用 `job_rdma.yaml` 创建 PyTorchTrainingJob 以启动训练，你可以如下修改训练配置：
+使用 `job.yaml` 创建 PyTorchTrainingJob 以启动训练，您可以如下修改训练配置：
 
 * 如要使用队列，取消第 6-9 行的注释，并修改第 8 行的队列名称（默认为 `default`）。
-* （`master` 之外的）工作器数量在第 68 行定义（默认为 `7`，加上 `master` 共 8 个工作器）。
-* 如要所有工作器尽可能地分配到同一个节点上，取消第 18-26、74-82 行的注释。
+* 工作器数量在第 12 行定义（默认为 `8`）。
+
+`job.yaml` 有如下变体，您也可以使用这些配置文件来创建 PyTorchTrainingJob：
```

Comment:
We need to provide a set of good + common variants. These variant are to be discussed/improved in later PRs.